gap_hci.go

```diff
@@ -427,6 +431,46 @@ func (a *Advertisement) Start() error {
 				}
 			}
 
+			switch {
+			case a.adapter.hci.connectData.connected:
+				random := a.adapter.hci.connectData.peerBdaddrType == 0x01
```

Comment:
Magic number?

Reply:
That magic number also appears in the `Scan()` function. I will correct that in a future to reduce the number of merge conflicts.